front/pages/edit.vue

```diff
@@ -95,6 +146,21 @@
         :open="openSuccessEditModal"
         >La couleur a été changée avec succès !</AppAlert
       >
+      <AppAlert
+        variant="info"
+        @close="showHelp = false"
+        :open="showHelp"
+      ><div>
+        <ul>
+          <li> Le drapeau est divisé en autant de zone de tailles égales qu'il y a de joueurs. + de joueurs = + de zones un peu plus petites</li>
+          <li> Chaque joueur controle la couleur de sa zone. Tu ne peux modifier la couleur de ta zone que 1 fois toutes les {{ maxCooldownTime }} minutes</li>
+          <li>Tu peux voir les pseudos, la couleur et les coordonnées x:y de tes voisins directs.
+          </li>
+          <li>Si tu cliques sur le bouton avec tes coordonnées et ton nom, ta zone sera mise en évidence</li>
+          <li><a target="_blanks" href="https://discord.gg/NTCwY6Cv">Élabores un plan sur discord pour dessiner un <strike>pén</strike>...un soleil en cliquant ici</a></li>
+        </ul>
```

Comment:
Ending punctuation is not consistent, please consider adding missing trailing dots.

Reply:
Looks like you missed that one